src/content/resources/coming-from/js-to-dart.md

```diff
@@ -2775,11 +2775,12 @@ Some possible next steps:
 
 * [Language tour][] to learn more about the Dart language
 * [Core library documentation][] to learn about Dart's core libraries
-* [Dart codelabs][] for hands-on experience learning a variety of topics
+* [Dart codelabs and tutorials][] for hands-on experience
```

Comment:
**issue:** Just call them _tutorials_. Codelabs is a Google tool, not a type of documentation. It will keep terms consistent throughout this PR and the corpus.

```suggestion
* [Dart tutorials][] for hands-on experience
```

Apply this wherever else it's appropriate.